samcli/lib/build/app_builder.py

```diff
@@ -413,6 +413,8 @@ def _build_lambda_image(self, function_name: str, metadata: Dict, architecture:
             "buildargs": docker_build_args,
             "decode": True,
             "platform": get_docker_platform(architecture),
+            "rm": True,
+            "forcerm": True,
```

Comment:
I see that here we have `rm` True but not `forcerm`: https://github.com/aws/aws-sam-cli/blob/develop/samcli/local/docker/lambda_image.py#L266
Should we keep them consistent?

Reply:
actually I was hesitated to add `forcerm`, I will remove it.